inscriptions/ISic003454.xml

```diff
@@ -194,14 +199,25 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#MB">
+                <ab>
+                    <lb n="1"/><g ref="#crux"/> Ένθάδε
+                    <lb n="2"/>κὲτε Ἰωά
+                    <lb n="3" break="no"/><supplied reason="lost">νν</supplied>ης μον
+                    <lb n="4" break="no"/><supplied reason="lost">αχὸ </supplied>ς ὁ Με
```

Comment:
close up the space after αχὸ, and tag Iwannes to Menouth as persName, with name elements on Ioannes and Menouth